src/js/lib/content/src/templates.ts

```diff
@@ -0,0 +1,245 @@
+/// This module defines helper methods for templates view.
+
+import { ProjectManager } from './project_manager'
+
+const PM = ProjectManager.default()
+
+const PROJECTS_LIST = 'projects-list'
+const PROJECTS_LIST_NEW_PROJECT = 'projects-list-new-project'
+
+const CARD_SPREADSHEETS = 'card-spreadsheets'
+const CARD_GEO = 'card-geo'
+const CARD_VISUALIZE = 'card-visualize'
+const CARD_BMW_DRIVERS = 'card-bmw-drivers'
+
+const ALL_CARDS = [
+    CARD_SPREADSHEETS,
+    CARD_GEO,
+    CARD_VISUALIZE,
+    CARD_BMW_DRIVERS
+]
+
+/**
+ * The sore for hidden elements.
+ *
+ * When the templates view is loaded, it hides some top-level elements, as
+ * their style is messing up scrolling. Hidden elements will be restored before
+ * loading the IDE.
+ */
+let hiddenElements: HTMLDivElement[] = []
+
+/** Status box div element for displaying errors. */
+let statusBox: HTMLElement = undefined
```

Comment:
variable name is "status box" and docs starts with "status box" - this is unnecessary repetition. I mean, I know that "statusBox" is a "status box" :) Just keep the docs short and with added info please

Reply:
Also, do you need to create global mutable variables? Could you just pass them and create them in local functions instead?